source/frontend/src/features/properties/map/project/hooks/useAddProjectFormManagement.tsx

```diff
@@ -14,6 +16,13 @@ export interface IUseAddProjectFormProps {
 export function useAddProjectForm(props: IUseAddProjectFormProps) {
   const { onSuccess } = props;
   const { addProject } = useProjectProvider();
+  const { getPublicByType } = useLookupCodeHelpers();
+  const initialValues = props.initialForm ?? new ProjectForm();
+  if (!initialValues.projectStatusType) {
```

Comment:
want to avoid fully hardcoding this in case the 'AC' value is removed or disabled.

Reply:
Even better would be to have the status types as enums